x-pack/plugins/maps/public/actions/map_actions.ts

```diff
@@ -176,6 +176,7 @@ export function mapExtentChanged(mapExtentState: MapExtentState) {
       mapState: {
         ...dataFilters,
         ...mapExtentState,
+        isForceRefresh: false,
```

Comment:
isForceRefresh does not exist on mapState (MapContext type) and should not added here. If you look at the MAP_EXTENT_CHANGED reducer its implemenation is 
```
case MAP_EXTENT_CHANGED:
      const newMapState = {
        center: action.mapState.center,
        zoom: action.mapState.zoom,
        extent: action.mapState.extent,
        buffer: action.mapState.buffer,
      };
      return { ...state, mapState: { ...state.mapState, ...newMapState } };
```
so really the only thing that is getting used by the reducer is center, zoom, extent, and buffer

Reply:
Thanks, discussed offline.

This is highlighting two issues:
- isForceRefresh should not be stored in the store-state. e.g. the map-extent change _does_ have to be considered as `isForceRefresh: false`. Putting this in the store opens the door for all kinds of race conditions. Instead, we'll trat this flag similar to how `isForceRefreshDueToDrawing` is handled, as an explicit param to the `syncData`-methods
- mixin in the data-filters introduces unnecessary cruft, and obscures the fact that the `map_extent_changed`-reducer only updates map-extent-related properties. Will change this in another PR, as the problem is unrelated.